gubernator/view_build.py

```diff
@@ -71,8 +71,9 @@ def parse_xml(self, xml, filename):
             try:
                 tree = ET.fromstring(re.sub(r'[\x00\x80-\xFF]+', '?', xml))
             except ET.ParseError, e:
-                self.failed.append(
-                    ('Gubernator Internal Fatal XML Parse Error', 0.0, str(e), filename, ''))
+                if re.match(r'junit.*\.xml', os.path.basename(filename)):
```

Comment:
@rmmh left the errors in this case so we don't lose that signal for existing jUnit producers

Reply:
can you do the same for the else case below? `logging.error('unable to find failures, unexpected tag %s', tree.tag)`

Reply:
Ah yes sorry, missed that one